pkg/build/builder/dockerutil_test.go

```diff
@@ -214,3 +218,69 @@ func TestCGroupParentExtraction(t *testing.T) {
 		}
 	}
 }
+
+func writeFile(fileName, fileContent string, t *testing.T) {
+	file, err := os.Create(fileName)
+	if err != nil {
+		t.Errorf("error creating file %s: %s", fileName, err.Error())
+	}
+	defer file.Close()
+	err = ioutil.WriteFile(fileName, []byte(fileContent), 0777)
```

Comment:
`0777` looks a bit excessive. Perhaps
```suggestion
	err = ioutil.WriteFile(fileName, []byte(fileContent), 0o644)
```